### PR TITLE
Upgrade/django 3.2

### PR DESCRIPTION
**Pull request by VirginiaDooley**
This work upgrades YNR to Django 3.0 and updates site packages accordingly.

### CODE REVIEW

File: ynr/settings/base.py [hunk 1]
```diff
@@ -212,7 +212,7 @@ def root(*path):
 INTERNAL_IPS = ["127.0.0.1"]
 
 # Language settings (calculated above):
-LANGUAGES = [("en", "English")]
+LANGUAGES = [("en-gb", "British English")]
```
Comment (michaeljcollinsuk) [inline ynr/settings/base.py:215]: Were this change required due to an upgrade?

Comment (VirginiaDooley) [reply ynr/settings/base.py:215]: Yes, I believe it was the result of an error related to the django 3.0 upgrade.
